fe/fe-core/src/test/resources/conf/cluster_snapshot.yaml

```diff
@@ -0,0 +1,39 @@
+# do not include leader fe
+frontends:
```

Comment:
FE configure file is xml or json，why invole a new config type ?

Reply:
using yaml is designed by pm ...